<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WRobotSettingMenuClass</class>
 <widget class="QWidget" name="WRobotSettingMenuClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WRobotSettingMenu</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>150</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>150</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	background-color: #CEDBF3;
	border-radius:10px
}

QLabel{
	background-color: rgb(0, 0, 0);
}

QPushButton{
	font: 14px &quot;微软雅黑&quot;;
	background-color: transparent;
	text-align:left;
	padding-left:12px;
	border:none;
	border-bottom:1px solid #000000
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pb_system">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>System Settings</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_chatbot">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Chatbot Settings</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_quite">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>√Quite Floating Window</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
